src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -629,21 +633,24 @@ public void testPrintTreeOnMoreThanOneFile() throws Exception {
             assertEquals("", systemErr.getLog());
         });
 
-        Main.main("-t", getPath("checks/metrics"));
+        Main.main("-t", getNonPackagePath(
+                "checks/metrics"));
```

Comment:
I did not understood. I should provide new method or cope files from those directory?